src/components/Carousel/index.js

```diff
@@ -3,6 +3,8 @@ import PropTypes from 'prop-types'
 import CarouselControl from './CarouselControl'
 import CarouselImages from './CarouselImages'
 import CarouselItem from './CarouselItem'
+import Arrow from '../../../prototype/global-images/arrow.svg'
+import { breakpoint } from '../../theme'
```

Comment:
I don't think we use this here, was there a desire to?

Reply:
I think that was added when we were trying some stuff out, we can remove that line